e2e-tests/pitr2/run

```diff
@@ -0,0 +1,383 @@
+#!/bin/bash
```

Comment:
I propose to call this test `pitr-gap-errors` since we are testing errors and issue when gap happens.
Other name could be `pitr-gap-invalid-binlogs`.
One thing which we might consider is do we really need to test some of these errors in E2E test, maybe unit test or integration test could be enough.

Reply:
As I understand, it is not easy to test it using the unit test. I agree with you that we need to rename the test.

Reply:
it was done [4a5ede8](https://github.com/percona/percona-xtradb-cluster-operator/pull/1423/commits/4a5ede80545bbb17a4b6f1d7e1d46ebf57265d7b)

Reply:
I don't it's possible to cover this in unit/integration test. We would need to mock stuff but that would not be enough to test this scenario.